glean/metrics.yaml

```diff
@@ -1325,3 +1325,165 @@ sample:
         description: |
           The id of the step that was viewed
         type: string
+
+  # Start of push messages validation metrics
+  websocket_connection_attempted:
+    type: event
+    lifetime: ping
+    send_in_pings:
+      - main
+    description: |
+      Connection attempt has started.
+    bugs:
+      - https://github.com/mozilla-mobile/mozilla-vpn-client/issues/3982
+    data_reviews:
+      - https://github.com/mozilla-mobile/mozilla-vpn-client/pull/4275#issuecomment-1228532390
+    data_sensitivity:
+      - technical
+    notification_emails:
+      - brizental@mozilla.com
+    expires: 2022-12-31
+
+  websocket_connected:
+    type: event
+    lifetime: ping
+    send_in_pings:
+      - main
+    description: |
+      Connection was succesfull.
+    bugs:
+      - https://github.com/mozilla-mobile/mozilla-vpn-client/issues/3982
+    data_reviews:
+      - https://github.com/mozilla-mobile/mozilla-vpn-client/pull/4275#issuecomment-1228532390
+    data_sensitivity:
+      - technical
+    notification_emails:
+      - brizental@mozilla.com
+    expires: 2022-12-31
+
+  websocket_errored:
+    type: event
+    lifetime: ping
+    send_in_pings:
+      - main
+    description: |
+      There was an error with the connection.
+    bugs:
+      - https://github.com/mozilla-mobile/mozilla-vpn-client/issues/3982
+    data_reviews:
+      - https://github.com/mozilla-mobile/mozilla-vpn-client/pull/4275#issuecomment-1228532390
+    data_sensitivity:
+      - technical
+    notification_emails:
+      - brizental@mozilla.com
+    expires: 2022-12-31
```

Comment:
maybe this should not expire. What do you think?

Reply:
I added the expiry because I think these metrics don't need to be here indefinitely, maybe a subset of them, but not all. I will get an email when they are about to expire and by then I should know which are the keepers and then I can go ahead and change this. WDYT?